metadata.json

```diff
@@ -15,6 +15,10 @@
     {
       "name": "puppetlabs/inifile",
       "version_requirement": ">= 1.4.1 < 5.0.0"
+    },
+    {
+      "name": "puppetlabs/puppetserver_gem",
+      "version:requirement": ">= 1.0.0 < 2.0.0"
```

Comment:
Syntax looks wrong. Should use underscore, not colon?

Reply:
Otherwise +1 for this change.  Not since a long time has the been a soft dependency.  At one point we supported non-puppetserver based masters.